docs/manifest/introduction.rst

```diff
@@ -0,0 +1,142 @@
+Introduction
+============
+
+The pyATS Manifest is a file with YAML syntax describing how and where to execute a script.
+It is intended to formally describe the execution of a single script, the runtime environment,
+script arguments and profile which includes the test topology and related parameters.
+
+A script can be executed via the manifest using the `pyats run manifest` command. Manifest
+files typically use the file extension ``.tem`` which stands for Test Execution Manifest.
+
+Command line execution examples:
+
+.. code-block:: shell
+
+    $ pyats run manifest job.tem
+    $ pyats run manifest job.tem --profile sanity
+
+Command line manifest validation:
+
+.. code-block:: shell
+
+    $ pyats validate manifest job.tem
+
+The manifest defines the script execution arguments, runtime environment and the profile(s)
+to define environment specific settings and arguments. The profiles are intended to allow
+the same script to be run against mulitple environments, e.g. local testbed, orchestrated
+testbed or production environment. Each profile can have its own specific script arguments
+and runtime specific settings.
+
+Script arguments
+~~~~~~~~~~~~~~~~
+
+One of the primary definitions in the manifest is the ``arguments`` key in the manifest.
+
+The arguments are defined as a set of key value pairs to be used as execution options.
+These arguments are translated to shell command arguments for script execution.
+
+Arguments defined for the script can be superseded by arguments defined in a profile.
+Any arguments specified on the command line will override arguments defined in the profile.
+
+The priority of arguments is: command line > profile > script.
+
+For example, the script arguments defined in the manifest could look like this:
+
+.. code-block:: yaml
+
+    arguments:
+        mail-html: True
+        configuration: easypy_config.yaml
+
+    profile:
+        local:
+            arguments:
```

Comment:
i think here it's good convention to produce a set of rules of how these yaml sections gets converted into your cli arguments

one that is apparent is anything will get prefixed with --.
what else?